packages/gatsby-transformer-remark/src/__tests__/on-node-create.js

```diff
@@ -93,13 +88,37 @@ Sed bibendum sem iaculis, pellentesque leo sed, imperdiet ante. Sed consequat ma
         },
         { excerpt_separator: `<!-- end -->` }
       ).then(() => {
-        expect(createNode.mock.calls).toMatchSnapshot()
-        expect(_.isString(createNode.mock.calls[0][0].excerpt)).toBeTruthy()
-        expect(createNode.mock.calls[0][0].excerpt).not.toEqual(0)
-        expect(createParentChildLink.mock.calls).toMatchSnapshot()
-        expect(createNode).toHaveBeenCalledTimes(1)
-        expect(createParentChildLink).toHaveBeenCalledTimes(1)
+        expect(actions.createNode.mock.calls).toMatchSnapshot()
+        expect(
+          _.isString(actions.createNode.mock.calls[0][0].excerpt)
+        ).toBeTruthy()
+        expect(actions.createNode.mock.calls[0][0].excerpt).not.toEqual(0)
+        expect(actions.createParentChildLink.mock.calls).toMatchSnapshot()
+        expect(actions.createNode).toHaveBeenCalledTimes(1)
+        expect(actions.createParentChildLink).toHaveBeenCalledTimes(1)
+      })
+    })
+  })
+
+  describe(`date formatting`, () => {
```

Comment:
Is it worth mentioning here (maybe in comment) that this is only needed because there is bug in gatsby core that doesn't really allow sorting mix of date object and strings?